zap/src/main/java/org/parosproxy/paros/extension/option/OptionsViewPanel.java

```diff
@@ -111,6 +111,7 @@ public int getValue() {
         }
     }
 
+    private static int s = 1;
```

Comment:
I don't mean to be picky but 6mo from now no one is going to remember what `s` is without digging through the code, could you make this `iconSize`? (Or something like that)